components/Clue/Clue.test.js

```diff
@@ -13,13 +13,13 @@ test("displays two different symbols", async () => {
 });
 
 test("displays randomly provided colors", async () => {
-  const randomColor = ["", "red", "blue"];
-  const randomSymbol = ["", "A", "B"];
-  render(<Clue randomColor={randomColor} randomSymbol={randomSymbol} />);
+  const randomColors = ["", "red", "blue"];
+  const randomSymbols = ["", "A", "B"];
+  render(<Clue randomColors={randomColors} randomSymbols={randomSymbols} />);
 
   // find all StyledColorDiv components and check their styles
   const colorDivs = screen.getAllByTestId("color-div");
   expect(colorDivs).toHaveLength(2);
-  expect(colorDivs[0]).toHaveStyle(`background-color: ${randomColor[1]};`);
-  expect(colorDivs[1]).toHaveStyle(`background-color: ${randomColor[2]};`);
+  expect(colorDivs[0]).toHaveStyle(`background-color: ${randomColors[1]};`);
+  expect(colorDivs[1]).toHaveStyle(`background-color: ${randomColors[2]};`);
 });
```

Comment:
Same here. I think `async` is not necessary.

Reply:
✅